packages/sdk/src/BoostCore.ts

```diff
@@ -394,28 +401,12 @@ export class BoostCore extends Deployable<
         isBase: isBase,
         instance: validator.address,
         parameters: isBase
-          ? validator
-              .buildParameters(
-                {
-                  signers: [owner],
-                  validatorCaller: coreAddress,
-                },
-                options,
-              )
-              .args.at(0) || zeroHash
+          ? validator.buildParameters(undefined, options).args.at(0) || zeroHash
```

Comment:
i think this'll be fine so long as we're the only validator, but i think technically the validator needs parameters to clone.